azDeploySecureSub.bicep

```diff
@@ -151,13 +154,16 @@ var secrets = [
   }
 ]
 
+// TODO: Consider renaming to resourceTypes
 var workloads = [
   'vnet'
   'st'
   'webApp'
   'kv'
   'mysql'
   'plan'
+  'uami'
+  'dplscr'
```

Comment:
There's a "hidden" dependency between these and the name variables (lines ~34 and onward) not sure what the best way to go forward is: either calling this out, or somehow enforce this programmatically if possible

Reply:
@matheuscarboni Agreed. I would prefer instead of an array if this was an object. My biggest concern is around the brittleness of the array indexers. We'll probably have to address it later. You can open an issue for it so it doesn't get lost.